snarkos/display/aleo.rs

```diff
@@ -85,7 +87,8 @@ pub fn notification_message<N: Network>(miner: Option<Address<N>>) -> String {
     .bold();
 
     if let Some(miner) = miner {
-        output += &format!(
+        let _ = write!(
```

Comment:
Does this need the `let _ = ` part?

Reply:
same reasoning as in my other comment; not having `let _ =` would cause an `unused Result that must be used` warning, and there is no possibility of an error (unless it's an OOM, in which case there are bigger issues), so the `Result` doesn't need to be handled